src/toil/jobStores/aws/jobStore.py

```diff
@@ -676,15 +723,16 @@ def get_shared_public_url(self, shared_file_name):
         self._requireValidSharedFileName(shared_file_name)
         return self.get_public_url(self._shared_file_id(shared_file_name))
 
-    def _connectSimpleDB(self):
+    def _connectSimpleDB(self) -> SimpleDBClient:
         """
-        :rtype: SDBConnection
+        :rtype: SimpleDBClient
         """
-        db = boto.sdb.connect_to_region(self.region)
-        if db is None:
+        sdb = boto3_session.client(service_name="sdb", region_name=self.region)
+        # db = boto.sdb.connect_to_region(self.region)
+        if sdb is None:
             raise ValueError("Could not connect to SimpleDB. Make sure '%s' is a valid SimpleDB region." % self.region)
-        monkeyPatchSdbConnection(db)
-        return db
+        # monkeyPatchSdbConnection(db)
+        return sdb
```

Comment:
Probably we should drop this function and just use `boto3_session.client()`, or a `self.sdb` set from `boto3_session.client()`. All it does is call a method and check for a None that I think can't happen anymore.